app/models/service_template.rb

```diff
@@ -400,8 +400,19 @@ def self.create_from_options(options)
   end
   private_class_method :create_from_options
 
-  def provision_request(user, options = nil)
-    provision_workflow(user, options).submit_request
+  def provision_request(user, options = nil, request_options = nil)
+    provision_workflow(user, options, request_options).submit_request
+  end
+
+  def provision_workflow(user, dialog_options = nil, request_options = nil)
```

Comment:
@gmcculloug It was one of bill's feedback to make the function public since its needed by the REST API